crates/pindakaas/src/helpers.rs

```diff
@@ -355,4 +387,88 @@ pub(crate) mod tests {
 			}
 		}
 	}
+
+	// TODO [?] Some unused code I don't know what to do with.
+	macro_rules! lit {
+		($lit:expr) => {
+			$crate::Lit(std::num::NonZeroI32::new($lit).unwrap())
+		};
+	}
+
+	// 	macro_rules! clause {
+	// 	($($x:expr),+ $(,)?) => {
+	//             &[$($crate::lit!($x)),+]
+	// 	};
+	// }
+
+	// 	macro_rules! clauses {
+	// 	($($x:expr),+ $(,)?) => {
+	//             &[$($crate::clause!($x)),+]
+	// 	};
+	// }
+
+	/// A const Cnf (constructed at compile-time)
+	/// TODO lits are assumed to be in a contiguous var range starting from 1..
+	#[derive(Debug)]
+	pub(crate) struct ConstCnf {
+		lits: &'static [Lit],
+		sizes: &'static [usize],
+	}
+
+	impl ConstCnf {
```

Comment:
This is unused for now, I was hoping to compile in all the CNF's generated by the espresso SCM method. However this never finishes compilation. This is definitely an important issue, perhaps to fix for this PR: 

1) Efficiently include all SCM data
2) Fallback method (Scm::Pow) does not cover everything (it might still if you stick to strictly PB, but definitely for integer problems and some configs it might fail)

Reply:
On top of this, I'm wondering if there is an easier way to create a const version of a struct containing Vec's?